doc/lua_api.txt

```diff
@@ -2548,6 +2548,9 @@ Some types may inherit styles from parent types.
 
 ### Valid Properties
 
+* box
+    * noclip - boolean, set to true to allow the element to exceed formspec bounds.
+        * Default to true in formspec_version version 3 or higher
```

Comment:
`style.getBool(StyleSpec::NOCLIP, m_formspec_version < 3)`

This means that _noclip_ defaults to `true` for versions _prior_ 3. Thus it will clip from version 3 onwards.